connexion/utils.py

```diff
@@ -137,7 +137,7 @@ def is_json_mimetype(mimetype):
     :rtype: bool
     """
     maintype, subtype = mimetype.split('/')  # type: str, str
```

Comment:
A MIME type has the following structure:
```
type/subtype;parameter=value
```
So instead of changing the check itself, we should extract the subtype in the correct way by accounting for the `;` and splitting on it

Reply:
Thanks for the feedback. I will make the update later this week.

Reply:
@Ruwann tests are now added and the proposed changes are done.